src/main/java/frc/trigon/robot/subsystems/roller/placeholderroller/PLACEHOLDERRollerConstants.java

```diff
@@ -0,0 +1,52 @@
+package frc.trigon.robot.subsystems.roller.placeholderroller;
+
+import com.ctre.phoenix6.StatusSignal;
+import com.ctre.phoenix6.configs.TalonFXConfiguration;
+import com.ctre.phoenix6.hardware.TalonFX;
+import com.ctre.phoenix6.signals.InvertedValue;
+import com.ctre.phoenix6.signals.NeutralModeValue;
+import edu.wpi.first.wpilibj.DigitalInput;
+import frc.trigon.robot.constants.RobotConstants;
+import frc.trigon.robot.motorsimulation.MotorSimulation;
+
+public class PLACEHOLDERRollerConstants {
+    private static final int
+            MOTOR_ID = 0,
+            INFRARED_SENSOR_CHANNEL = 0;
+    private static final NeutralModeValue MOTOR_NEUTRAL_MODE_VALUE = NeutralModeValue.Coast;
+    private static final InvertedValue MOTOR_INVERTED_VALUE = InvertedValue.CounterClockwise_Positive;
+    private static final double SENSOR_TO_MECHANISM_RATIO = 1;
```

Comment:
This can just be named GEAR_RATIO. Since this is a duplicate for both the sim and the real ios, have a constant for it in the RollerConstants